pkg/flare/archive.go

```diff
@@ -84,18 +87,25 @@ type filePermsInfo struct {
 	group string
 }
 
+// Profile is a performance profile containing heap and CPU profiles.
+type Profile struct {
+	FirstHeapProfile  []byte
+	CPUProfile        []byte
+	SecondHeapProfile []byte
```

Comment:
Should we store a slice of `[]byte` for the heap ? Do we have any use cases where we might want 3 or more heap profile ?

small nit: if we keep the 2 profile could we put them next to each other in the struct:

```go
type Profile struct {
￼	CPUProfile        []byte
￼	FirstHeapProfile  []byte
￼	SecondHeapProfile []byte
}
```

Reply:
It's only 2 profiles because one is created before the cpu profile, and one after the cpu profile to have a diff. Don't think we need more than that.

Next to each other makes sense 👍